src/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -149,4 +150,16 @@ class CSVUtils {
     static CSVPrinter getCSVPrinter() {
         return new CSVPrinter(new StringBuilder(), CSVFormat.DEFAULT)
     }
+
+    /**
+    * Return a CSV separator character based on provided data and expected number of column
+    * */
+    static char getSeparator(String text, Integer columNumber) {
+        def otherSeparator = ';'
```

Comment:
Perhaps you could add a const for this? Beside the `DEFAULT_COLUMN_SEPARATOR`, as a `CUSTOM_COLUMN_SEPARATOR`?